### ISSUE TITLE
DFL needs a bump due to complete breakage in new Steam Client

### ISSUE THREAD

**Issue by HugLifeTiZ** [SteamGridDB/decky-steamgriddb#104]
[See this specific comment](https://github.com/SteamDeckHomebrew/decky-loader/issues/597#issuecomment-1987138946) of SteamDeckHomebrew/decky-loader#597. Thanks for your time.

**Comment by doZennn** [comment]
SGDB uses global DFL so that's not the problem. The beta issues are coming from other components that i fixed a few days ago already. The new version of the plugin is just waiting to be merged into the store.

**Comment by HugLifeTiZ** [comment]
It's waiting to be merged? Is it waiting on someone on Decky's side? Considering every plugin broke completely in the new client, you'd think you wouldn't have to wait very long.
